lib/elixir/lib/module/types.ex

```diff
@@ -147,15 +147,22 @@ defmodule Module.Types do
 
   def error_to_warning(:unable_unify, {left, right, stack}, context) do
     {fun, arity} = context.function
-    location = {context.file, get_line(stack), {context.module, fun, arity}}
+    location = {context.file, get_position(stack), {context.module, fun, arity}}
```

Comment:
I think it is easier if we store the put the whole `meta` in here and then we handle it on the parallel checker only. :)